Celeste.Mod.mm/Patches/Strawberry.cs

```diff
@@ -27,5 +27,20 @@ public patch_Strawberry(EntityData data, Vector2 offset, EntityID gid)
             Everest.Discord.OnStrawberryCollect();
         }
 
+        public extern void orig_Update();
+        [PatchStrawberryTrainCollectionOrder]
+        public new void Update()
+        {
+            orig_Update();
+        }
+
+        /*[MonoModReplace]
+        private new bool IsFirstStrawberry
+        {
+            get
+            {
+                return StrawberryRegistry.IsFirstStrawberry(this);
+            }
+        }*/
```

Comment:
Any reason why this is here? 😅

Reply:
That was me trying and failing something else until I found out that, for some mysterious reason, the instruction I needed to find and replace was actually a callvirt instead of a call. Not sure how that even happened.

I'll go ahead and run down all the development breadcrumbs like this.